engine/src/main/java/com/redhat/cloud/notifications/templates/models/Environment.java

```diff
@@ -20,4 +20,20 @@ public String name() {
     public String url() {
         return this.url;
     }
+
+    /**
+     * Checks if the "ENV_NAME" environment variable is set to "local-dev".
+     * @return true if the "ENV_NAME" environment variable is set to "local-dev".
+     */
+    public boolean isEnvironmentLocal() {
+        return "local-dev".equals(this.environment);
+    }
+
+    /**
+     * Checks if the "ENV_NAME" environment variable is set to "stage".
+     * @return true if the "ENV_NAME" environment variable is set to "stage".
+     */
+    public boolean isEnvironmentStage() {
+        return "stage".equals(this.environment);
+    }
```

Comment:
This could probably be moved to `common`

Reply:
Good idea, I've moved it and updated the references! Thanks!